sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -359,10 +359,22 @@ def get_cache_manager(self, pipeline, create_if_absent=False):
     manager for the pipeline."""
     cache_manager = self._cache_managers.get(str(id(pipeline)), None)
     if not cache_manager and create_if_absent:
-      cache_dir = tempfile.mkdtemp(
-          suffix=str(id(pipeline)),
-          prefix='it-',
-          dir=os.environ.get('TEST_TMPDIR', None))
+      from apache_beam.runners.interactive import interactive_beam as ib
+      if ib.options.cache_root:
```

Comment:
Can you please add a TODO here too to handle the case when the path is started with "gs://"? You may now just raise a ValueError about GCS path not supported.